CHANGES.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+### Shadows TODO
```

Comment:
Remove `TODO`, move this to the `1.22` section, and also add a link to the new Sandcastle example (using the future cesiumjs.org base path).